components/mail/dovecot/dovecot.p5m

```diff
@@ -0,0 +1,1055 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Josef 'Jeff' Sipek <jeffpc@josefsipek.net>
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+<transform file path=usr/libexec/dovecot/.* -> default mode 0555>
+
+user gcos-field="Dovecot mail server" username="dovecot" group="dovecot" uid=72
+user gcos-field="Dovecot login user" username="dovenull" group="nogroup" uid=73
+group groupname="dovecot" gid=72
+
+file files/dovecot.xml path=lib/svc/manifest/network/imap/dovecot.xml
+file files/dovecot path=lib/svc/method/dovecot
+
+file path=etc/dovecot/README
```

Comment:
This README is not a full readme - it's simply a a note informing the admin what to do:

```
Configuration files go to this directory. See example configuration files in
/usr/share/doc/dovecot/example-config/
```